index.html

```diff
@@ -2368,6 +2373,98 @@ <h3>
           can get their games and apps rated with IARC</a>.
         </p>
       </section>
+      <section>
+        <h3>
+          <code>shortcuts</code> member
+        </h3>
+        <p>
+          The <dfn>shortcuts</dfn> member is an <a>array</a> of
+          <a>ShortcutItem</a>s that provide access to key tasks within a web
+          application. For example, they can be used to link directly to a
+          user’s timeline within a social media application or their recent
+          orders in an e-commerce context.
+        </p>
+        <p>
+          A developer may provide as many shortcuts as they like, but the user
+          agent and operating system will decide how many are presented to the
+          user. For this reason, it is recommended that developers order their
+          shortcuts by priority, with the most critical shortcuts appearing
+          first in the array.
+        </p>
+        <p>
+          The steps for <dfn>processing the <code>shortcuts</code> member</dfn>
+          are given by the following algorithm. The algorithm takes a
+          <a data-cite=
+          "WEBIDL#sequence-type">sequence</a>&lt;<a>ShortcutItem</a>&gt;
+          <var>shortcuts</var> as an argument. This algorithm returns an
+          <a data-cite=
+          "WEBIDL#sequence-type">sequence</a>&lt;<a>ShortcutItem</a>&gt;. For
+          each <var>shortcut</var> (<a>ShortcutItem</a>) in the sequence, set
+          <var>shortcut.icons</var> to the result of running <a>processing
+          `ImageResource` members</a> given <var>shortcut.icons</var> and
+          <var>manifest URL</var>.
+        </p>
+        <p>
+          A user agent MUST expose shortcuts via interactions that are
```

Comment:
These MUSTs should be SHOULDs. We can't mandate what the user agent does with its own (or the OS's) UI. From a spec point of view, we treat the UA and OS as a single entity --- if the UA wants to invent a whole new mechanism for displaying the icon that's different to how the OS normally does it, well that's fine. It could also display these things inside the browser UI, not the OS at all.

We also can't impose a MUST on the ordering; the user agent could provide the list of shortcuts to the OS and then the OS may sort them alphabetically, or sort them based on how often the user presses each one.